### PR TITLE
Add release notes for 2.7.7

### PR DESCRIPTION
**Pull request by madkinsz**
[Rendered](https://github.com/PrefectHQ/prefect/blob/release-notes-2.7.6/RELEASE-NOTES.md#release-277)

Using this in the highlight:

Cloud API Reference Documentation

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.7.5...2.7.7
+
+
 ## Release 2.7.6
 
 This release fixes a critical bug in the SQLite database migrations in 2.7.4 and 2.7.5.
 
 See https://github.com/PrefectHQ/prefect/issues/8058 for details.
 
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.7.4...2.7.5
```
Comment (discdiver) [inline RELEASE-NOTES.md:50]:
```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.7.5...2.7.6
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
+- Add new REST API reference using Redoc — https://github.com/PrefectHQ/prefect/pull/7503
+
+### Contributors
+* @j-tr made their first contribution in https://github.com/PrefectHQ/prefect/pull/8013
+* @toby-coleman made their first contribution in https://github.com/PrefectHQ/prefect/pull/8083
+* @riquelmev made their first contribution in https://github.com/PrefectHQ/prefect/pull/7768
+* @joelluijmes
+
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.7.5...2.7.7
```
Comment (discdiver) [inline RELEASE-NOTES.md:41]:
```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.7.6...2.7.7
```

**Review by discdiver — APPROVED** [review]
lgtm other than "All changes" version numbers noted. Reviewed the content in the release notes - didn't look if anything was missing.

**Review by tpdorsey — APPROVED** [review]

File: RELEASE-NOTES.md (hunk 2 above)
Comment (madkinsz) [reply RELEASE-NOTES.md:54]: This one is actually intentional because 2.7.6 was a weird release.

**Pull request merged**